bot/src/commands/revert-last-command.js

```diff
@@ -0,0 +1,54 @@
+import { get, post } from '../utils';
+import { Commands } from './index';
+
+function getCommandByName(name) {
```

Comment:
If you change commands' names to match the names of the exports (or write a dash-to-pascal-case helper), you can do:

    import * as commands from './index';

and:

    return commands[name];